csharp/src/Microsoft.ML.OnnxRuntime/NamedOnnxValue.shared.cs

```diff
@@ -8,6 +8,11 @@
 using System.Diagnostics;
 using System.Linq;
 
```

Comment:
This class should be deprecated. We should not add any more changes where.

Reply:
Removed